CONTRIBUTING.md

```diff
@@ -49,32 +56,46 @@ Subscribe to all the MQTT topics the `yggd` client will publish and subscribe to
 to monitor the MQTT traffic.
 
 ```
-sub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/#
+sub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/#
 ```
 
 ### Terminal 4
 
 Publish a "PING" command to the `yggd` "control/in" topic.
 
 ```
-go run ./cmd/yggctl generate control-message --type command '{"command":"ping"}' | pub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/control/in
+go run ./cmd/yggctl generate control-message --type command '{"command":"ping"}' | \
+    pub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/control/in
 ```
 
 You should see the message logged to the output of `sub` in [Terminal
 3](#terminal-3) and receipt of the message logged in the output of `yggd` in
-[Terminal 1](#terminal-1).
+[Terminal 1](#terminal-1). The `echo` worker does not receive any message in this
```

Comment:
I don't think it makes sense to declare what _won't_ happen. If that were necessary, we would have to document what won't happen a lot, in many places. My guess is that you thought the ping would send a message to the echo worker; so perhaps we should reorder these sections to send the ping command before starting the echo worker. This shows that MQTT is working, without confusing things by starting the echo worker first.

Reply:
I tried the reordering of terminals and it would require switching of terminals. I will add more subsections to section `Terminal 4` to emphasize that we send only Ping command and we test sending MQTT command to `yggd`.